### PR TITLE
www-client/firefox: Fixes oudated nss dependency and adds sqlite dependency for +system-sqlite

### PR DESCRIPTION
**Pull request by tgbugs**
Gentoo-Bug: https://bugs.gentoo.org/590424

Fixes oudated nss dependency and adds +system-sqlite
dependent dependency check.

### CODE REVIEW

**Comment by mgorny** [comment]
Please use the summary line to summarize your changes so that people looking at the shortlog can see what changed instead of having to look bugs up and/or expand the commit message.

File: www-client/firefox/firefox-48.0.ebuild [hunk 1]
```diff
@@ -54,11 +54,12 @@ SRC_URI="${SRC_URI}
 ASM_DEPEND=">=dev-lang/yasm-1.1"
 
 RDEPEND="
-	>=dev-libs/nss-3.23
+	>=dev-libs/nss-3.24
 	>=dev-libs/nspr-4.12
 	selinux? ( sec-policy/selinux-mozilla )"
 
 DEPEND="${RDEPEND}
+	system-sqlite? ( >=dev-db/sqlite-3.12.2 )
```
Comment (mgorny) [inline www-client/firefox/firefox-48.0.ebuild]: Please try to keep things sorted. I think the arch-related flags were intentionally listed last but other flags should be sorted lexically, i.e. system-sqlite after pgo.

Comment (tgbugs) [inline www-client/firefox/firefox-48.0.ebuild]: Thanks, fixed.

**Comment by tgbugs** [comment]
Fixed by https://github.com/gentoo/gentoo/commit/66126cdffdddccccb9c8997b03b67d808767425e

**Pull request closed without merging**